config/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "development": {
     "username": "root",
-    "password": "password",
+    "password": "teddy",
```

Comment:
STOP CHECKING IN THIS PASSWORD PLEASE.

Reply:
Your frontend work has nothing to do with changing the configuration for the database, please refrain and listen to me, do not change this.

Reply:
Ok. then what do you need me to do to approve the merge?

Reply:
just use 'password' across all. this is frustrating. thanks.